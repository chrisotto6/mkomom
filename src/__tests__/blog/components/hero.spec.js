import React from 'react'
import { create } from 'react-test-renderer'
import Hero from '../../../blog/components/Hero'

describe('Hero', () => {
  it('renders correctly', () => {
    const tree = create(
      <Hero category="test" title="test title" author="Chris Otto" date="10-06-1988" />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
