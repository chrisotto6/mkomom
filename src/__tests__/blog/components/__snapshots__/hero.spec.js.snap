// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders correctly 1`] = `
<div
  className="bg-washed-green ph2 pv5 flex flex-column justify-center items-center"
>
  <a
    className="sans-serif ttu mid-gray tracked f6"
    href="/test"
  >
    test
  </a>
  <h1
    className="dark-gray display fw4 f1-l f2 tc"
  >
    test title
  </h1>
  <span
    className="sans-serif tracked ttu f6 mb2"
  >
    by 
    Chris Otto
  </span>
  <span
    className="sans-serif tracked ttu f6"
  >
    10-06-1988
  </span>
</div>
`;
